BUILD_IMAGE_VERSION

```diff
@@ -0,0 +1 @@
+scanner-build-0.3.35-5-ge540580ff7
```

Comment:
This version is based on this commit: https://github.com/stackrox/rox-ci-image/commit/e540580ff70f1dfb61b176dd09d1a29ca090a37e

Reply:
will you tag a 0.3.36?